x-pack/plugins/endpoint/common/types.ts

```diff
@@ -168,39 +168,44 @@ export type AlertEvent = Immutable<{
     module: string;
     type: string;
   };
+  endpoint: {
+    policy: {
+      id: string;
+    };
+  };
   process: {
     code_signature: {
       subject_name: string;
       trusted: boolean;
     };
-    command_line: string;
-    domain: string;
+    command_line?: string;
```

Comment:
Is command_line actually an optional field? Or does it come back with every process? IMO shouldn't be optional unless it's actually optional. If it's breaking the type checker, then it can probably be reconciled somewhere upstream.

Reply:
I was marking it as optional from the UI perspective - it isn't used by the UI so making it required right now just adds extra work when making fake data.  I'm not sure if it is guaranteed to be in every returned process.